### PR TITLE
Update ind.js

### PR DESCRIPTION
**Pull request by athena123**
Market index contract should not have a expiry field. Having it is confusing.

### CODE REVIEW

File: lib/contract/ind.js [hunk 1]
```diff
@@ -2,14 +2,13 @@ var assert = require('assert');
 
 var _ = require('lodash');
 
-module.exports = function (symbol, expiry, currency, exchange) {
+module.exports = function (symbol, currency, exchange) {
   assert(_.isString(symbol), 'Symbol must be a string.');
   assert(_.isString(expiry), 'Expiry must be a string.');
```
Comment (mvberg) [inline lib/contract/ind.js]: @athena123 this assertion will throw now that expiry has been removed.

Comment (athena123) [reply lib/contract/ind.js]: you are right. thanks.

**Comment by meteorpublish** [comment]
:+1:, please merge. Indexes don't have expiry. Try for example "RUT".

**Comment by pilwon** [comment]
@meteorpublish @mvberg Merged and published as v0.2.7. Thanks!

**Pull request merged**